features/tasks/list_task.feature

```diff
@@ -0,0 +1,68 @@
+@domain @api
+Feature: Listing a task for a project
```

Comment:
Purely language, but I tend to think of "listing" things as well, lists. Since this is about pulling a single task for a project I might call it something like "Finding a task for a project" or something like that.